lib/kinetic_cafe/error_dsl.rb

```diff
@@ -141,27 +144,54 @@ def define_error(options)
       status ||= defined?(Rack::Utils) && :bad_request || 400
       status.freeze
 
-      severity ||= :error
+      if status
+        error.send :define_method, :default_status, -> { status }
+        error.send :private, :default_status
+        error.send :define_singleton_method, :default_status, -> { status }
+      end
+
+      severity ||= @severity || :error
       severity.freeze
 
-      error.send :define_method, :default_status, -> { status } if status
-      error.send :define_method, :default_severity, -> { severity } if severity
-      error.send :private, :default_status
+      if severity
+        error.send :define_method, :default_severity, -> { severity }
+        if severity
+          error.send :define_singleton_method, :default_severity, -> { severity }
+        end
+      end
+
+      error.instance_exec(&block) if block
 
       const_set(error_name, error)
     end
 
-    ##
-    def self.included(_mod)
+    # Override the default severity with +value+ for errors defined in
+    # the provided +block+.
```

Comment:
`# the provided +block+. Then, execute the block using the new default value for +@severity+ and change +@severity+ back to its old value` to keep the integrity of the class.`. Or something like that. Thoughts?